src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1792,8 +1792,7 @@ public SVNAuthentication createSVNAuthentication(String kind) {
 
             @Override
             public StandardCredentials toCredentials(String description) {
-                return new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, null, description, userName,
-                        getPassword());
+                return new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, null, description, userName, getPassword());
```

Comment:
@oleg-nenashev I'm not going to discuss if `line length` should o not be defined in a Coding Style Guide